stellar_sdk/sep/stellar_web_authentication.py

```diff
@@ -35,6 +35,37 @@
 MUXED_ACCOUNT_STARTING_LETTER: str = "M"
 
 
+class ChallengeTransaction:
```

Comment:
Hi @JakeUrban, do you have any suggestions for the name of this class?

Reply:
I think `ChallengeTransaction` is sufficient.